src/hci.cpp

```diff
@@ -1189,14 +1189,14 @@ INT_RETVAL intRunWidgets()
 	{
 		SecondaryWindowUp = true;
 		intRunDesign();
+	} else if (intMode == INT_INTELMAP) {
```

Comment:
I would prefer the `else if` be on its own line.

Reply:
While we are at it, why Warzone uses new lines for braces and `else if`? I think nearly every C++ project places them in the same line. An every course / example I have ever seen. The only language that uses "braces in new line" is C#.

Reply:
> While we are at it, why Warzone uses new lines for braces and `else if`? I think nearly every C++ project places them in the same line. An every course / example I have ever seen. The only language that I know that uses "braces in new line" is C#.

This was the original code style (the code originated in the 1990s as C).

At some point I (personally) hope to re-format it, but the diff will be atrocious (and make comparing before and after said diff an absolute nightmare), so for now we're asking people to try to maintain the current code style in whatever file you're editing.

Reply:
Yeah, mostly for [consistency](https://github.com/Warzone2100/warzone2100/blob/master/doc/CodingStyle.md).